sesheta/utils.py

```diff
@@ -49,6 +49,7 @@
     "sub-mod": "Subin Modeel",
     "CermakM": "Marek Cermak",
     "vpavlin": "Vaclav Pavlin",
+    "tumido": "Tomáš Coufal",
```

Comment:
we can disable sesheta on the repos, as its workload would be handled by it successor sefkhet-abwy.
can you add this change here:
https://github.com/AICoE/Sefkhet-Abwy/blob/e7959ca0389ce8214cd2b271315c651989d9ce4a/aicoe/sesheta/utils.py#L40

Reply:
Ah, ok. Thanks for the review :+1: